_pages/robot/2013.html

```diff
@@ -0,0 +1,37 @@
+---
+layout: default
+title: 2013 Robot
+permalink: /robot/2013
+stylesheets:
+- banners
+description: Our robot and the challenge for the 2013 season
+---
+<main>
+    <h1>2013 Robot</h1>
+    <!-- {% include banners.html year=2013 %} We earned no banners in 2013-->
+    <img src="/images/robots/2013/2013-d.png" alt="Our 2013 robot (side view).">
+    <p>The FIRST challenge for this year was called Ultimate Ascent. The challenge included throwing frisbees towards various targets.  Points could be scored by throwing frisbees and climbing. For more information, see the <a href="https://www.youtube.com/watch?v=itHNW2OFr4Y">official game animation</a>.</p>
+    <h5>Design</h5>
+    <p>The 2013 robot was a square robot. This was chosen because it was easier to fit all the needed components inside. Also, the square frame made it easier to balance while climbing. The frame was mainly made out of C-Channel. The drive system used was tank drive. This was chosen to maximize power because the challenge required a fast and strong robot. </p>
+    <h5>Throwing</h5>
+    <p>Frisbees were first fed into a slot, and then transferred into the throwing system.  The throwing system consisted of 2 pneumatic tires, placed on either side, which were powered by CIM motors. Code was used to adjust the angle to score in different levels of targets.  Pre-coded to use 3 buttons on the joystick, the power percentage was set in order to shoot into all of the scoring targets. </p>
+    <h5>Intake</h5>
+    <p> To pick up a frisbee, a suction cup was connected onto a rotating pivot arm which connected to a piston. After picking up the frisbee on the floor, it was rotated 180° then dropped onto a shallow, circular, collection container. </p>
+    <div class="img-row">
+        <img src="/images/robots/2013/2013-a.jpg" alt="The 2013 robot picking up a frisbee.">
+        <img src="/images/robots/2013/2013-b.jpg" alt="The 2013 
+        \robot about to throw a frisbee.">
+        <img src="/images/robots/2013/2013-c.jpg" alt="The 2013 robot about to climb.">
+
+       
+    </div>
+    <h5>Climbing</h5>
+    <p>This year's robot could only climb as high as the first rung.  The climbing hook was made out of 80/20 and extended via piston.  To climb, the robot drove into the rung and then retracted the piston. </p>
+    <h5>Programming</h5>
+    <p>  The programming was mainly written in LabVIEW, a graphical language based on C, C++, and C#.    </p>
```

Comment:
Nit: clean up the spacing

Reply:
There's still leading and trailing space inside tags in both files - also would recommend replacing the double spaces after periods with one. Very minor issue, great job on this!